### PR TITLE
CONCF-519 | Implement Qualaroo in Mercury

### PR DESCRIPTION
**Pull request by vforge**
Ping @bognix @rogatty @Warkot

### CODE REVIEW

File: server/facets/showArticle.ts [hunk 1]
```diff
@@ -69,10 +69,19 @@ function onArticleResponse (request: Hapi.Request, reply: any, error: any, resul
...
+		if (!result.queryParams.noexternals) {
+			// optimizely
+			if (localSettings.optimizely.enabled) {
+				result.optimizelyScript = localSettings.optimizely.scriptPath +
+					(localSettings.environment === Utils.Environment.Prod ?
+						localSettings.optimizely.account : localSettings.optimizely.devAccount) + '.js';
+			}
+
+			// qualaroo
+			if (localSettings.optimizely.enabled) {
```
Comment (rogatty) [inline server/facets/showArticle.ts]: Should be `qualaroo.enabled`

File: server/views/_layouts/ember-main.hbs [hunk 2]
```diff
@@ -104,6 +104,13 @@
 			<script src="/front/scripts/main.js"></script>
 		<!-- endbuild -->
 
+		{{#if qualarooScript}}
+			<script>
+				var _kiq = _kiq || [];
+			</script>
+			<script src="{{qualarooScript}}" async="true"></script>
+		{{/if}}
```
Comment (rogatty) [inline server/views/_layouts/ember-main.hbs]: How about extracting this to a partial like `end-of-body-scripts.hbs`?

**Comment by rogatty** [comment]
LGTM

**Comment by Warkot** [comment]
LGTM :water_buffalo:

**Comment by bognix** [comment]
LGTM

**Pull request merged**